src/components/utdragFraSykefravaeret/UtdragFraSykefravaeret.tsx

```diff
@@ -231,39 +40,19 @@ export const Samtalereferat = () => {
 };
 
 const UtdragFraSykefravaeret = () => {
-  const { sykmeldinger } = useSykmeldingerQuery();
-  const { latestOppfolgingstilfelle } = useOppfolgingstilfellePersonQuery();
-
-  const innsendteSykmeldingerUtenArbeidsgiver =
-    sykmeldingerUtenArbeidsgiver(sykmeldinger);
-  const sykmeldingerIOppfolgingstilfellet =
-    sykmeldingerInnenforOppfolgingstilfelle(
-      innsendteSykmeldingerUtenArbeidsgiver,
-      latestOppfolgingstilfelle
-    );
-  const sykmeldingerSortertPaSykmeldingsperiode =
-    sykmeldingerSortertNyestTilEldstPeriode(sykmeldingerIOppfolgingstilfellet);
-
   return (
-    <Panel className="mb-4 h-min">
+    <Box padding="4" background="surface-default" className="mb-4 h-min">
       <Heading level="2" size="medium" className="mb-4">
         {texts.header}
       </Heading>
       <UtdragOppfolgingsplaner />
-      <SykmeldingerForVirksomhet />
-      {sykmeldingerSortertPaSykmeldingsperiode?.length > 0 && (
-        <SykmeldingerUtenArbeidsgiver
-          sykmeldingerSortertPaUtstedelsesdato={
-            sykmeldingerSortertPaSykmeldingsperiode
-          }
-        />
-      )}
+      <Sykmeldinger />
```

Comment:
Vi flyttet alt som var relevant for sykmelding inn i en den nye `Sykmeldinger.tsx` komponenten.